modules/installation-aws-permissions.adoc

```diff
@@ -71,7 +71,6 @@ cluster, the IAM user requires the following permissions:
 * `ec2:CreateVpc`
 * `ec2:CreateVpcEndpoint`
 * `ec2:ModifySubnetAttribute`
-* `ec2:ModifyVpcAttribute`
```

Comment:
Why we remove this line here? Per install source code - https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/aws/permissions.go, we still need this permission.

Reply:
Thank you @jianlinliu. I've fixed this error. Will you please take another look?